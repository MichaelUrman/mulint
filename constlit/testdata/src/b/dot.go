// Package b also tests exceptions such as 1*time.Second but with dot imports
package b

import . "time"

var Tdot Duration = 1 // want "Replace `1` with `Nanosecond`"
var Udot Duration = 1 * Second
var Vdot = Second * 1
var Wdot = 3*Second + 1 // want "Replace `1` with `Nanosecond`"

var Adot int = 2 // OK; don't use Wed
