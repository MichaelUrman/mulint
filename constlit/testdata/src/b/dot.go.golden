// Package b also tests exceptions such as 1*time.Second but with dot imports
package b

import . "time"

var Tdot Duration = Nanosecond // want "Replace `1` with `Nanosecond`"
var Udot Duration = 1 * Second
var Vdot = Second * 1
var Wdot = 3*Second + Nanosecond // want "Replace `1` with `Nanosecond`"

var Adot int = 2 // OK; don't use Wed
